spel/kickstarts/ks.centos7.minimal.cfg

```diff
@@ -20,16 +20,16 @@ bootloader --location=mbr --driveorder=sda --append="crashkernel=auto"
 clearpart --drives sda --all
 zerombr
 
-part /boot --fstype=ext4 --asprimary --size=200 --ondrive=sda
+part /boot --fstype=ext4 --asprimary --size=512 --ondrive=sda
 part pv.008002 --grow --size=200 --ondrive=sda
 
 volgroup VolGroup00 --pesize=4096 pv.008002
 logvol /              --name=rootVol  --vgname=VolGroup00 --size=4096 --fstype=ext4
+logvol swap           --name=swapVol  --vgname=VolGroup00 --size=2048
 logvol /home          --name=homeVol  --vgname=VolGroup00 --size=1024  --fstype=ext4
 logvol /var           --name=varVol   --vgname=VolGroup00 --size=2048  --fstype=ext4
 logvol /var/log       --name=logVol   --vgname=VolGroup00 --size=2048  --fstype=ext4
-logvol /var/log/audit --name=auditVol --vgname=VolGroup00 --size=8192 --fstype=ext4
-logvol swap           --name=swapVol  --vgname=VolGroup00 --size=2048
+logvol /var/log/audit --name=auditVol --vgname=VolGroup00 --size=8704 --grow --fstype=ext4
```

Comment:
Aesthetically, I'd probably have left the `--size` value as `8192` ...since that's a nice, round (KiB-based) boundary.